packages/analytics/package.json

```diff
@@ -37,7 +39,7 @@
     "rollup-plugin-commonjs": "10.1.0",
     "rollup-plugin-json": "4.0.0",
     "rollup-plugin-node-resolve": "5.2.0",
-    "rollup-plugin-typescript2": "0.27.1",
+    "rollup-plugin-typescript2": "0.27.0",
```

Comment:
We should upgrade to 0.27.1 which is the current version.

Reply:
Something always seems to go wrong with the rebasing. Fixed.